.github/workflows/sync_templates.yml

```diff
@@ -4,12 +4,6 @@ on:
   push:
     branches:
       - main
-    paths:
-      - .github/sync.yml
-      - .github/ISSUE_TEMPLATE/*
-      - .github/ISSUE_TEMPLATE.md
-      - .github/PULL_REQUEST_TEMPLATE/*
-      - .github/PULL_REQUEST_TEMPLATE.md
```

Comment:
Run this action on every push to main rather than only when these files are changed. That way we can update which files get synced without having to also update this list.